.eslintignore

```diff
@@ -1,3 +1,4 @@
 node_modules/**
 dist/**
 coverage/**
+*.stories.js
```

Comment:
Why don't we want to lint the stories?

Reply:
I wish I could add a comment for this. but since we started using `alias` to avoid `from '../'` in favor of `from "grommet"` the linter started complaining about the fact that we don't have `grommet` inside node_modules.

Reply:
Can we do an `// eslint-disable-line` then, instead of ignoring the whole file?